tests/storage/cdi_upload/test_upload_virtctl.py

```
@@ -413,6 +413,46 @@ def test_virtctl_image_upload_dv_with_exist_pvc(
         )
 
 
+@pytest.mark.sno
+@pytest.mark.polarion("CNV-16270")
+@pytest.mark.usefixtures("primary_udn_for_upload")
+def test_virtctl_image_upload_dv_in_pudn_namespace(
+    admin_client,
+    udn_namespace_for_dv_upload,
+    download_image,
```

Ruff (0.15.15)

[warning] Unused function argument: `download_image`

(ARG001)